lib/internal/modules/esm/module_job.js

```diff
@@ -186,6 +195,20 @@ class ModuleJob {
     }
   }
 
+  runSync() {
+    assert(this.module instanceof ModuleWrap);
+    if (this.instantiated !== undefined) {
+      return { __proto__: null, module: this.module };
+    }
+
+    this.module.instantiate();
+    this.instantiated = PromiseResolve();
+    const timeout = -1;
```

Comment:
Just curious: is this merely to avoid a magic number?

Reply:
No idea, I've copied that from line 214.